### DISCUSSION TITLE
Tabs Widget - Missing the ability to style the active tab's hover color

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#21818] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The tabs widget lacks the ability to style/override the active tab's hover color style.

### Describe the solution you'd like

Provide what's lacking.

### Describe alternatives you've considered

Applying custom CSS.
![e-tab-active-on-hover](https://user-images.githubusercontent.com/35966122/228589373-51f8b06e-ccd3-4535-8116-9f956729402e.png)

**Comment by Nylash** [comment]
Did you manage to solve this with custom CSS ? I have the same issue :(

**Comment by Nylash** [comment]
Ok, I personnaly managed to do what I wanted, hide it. I simply had set the global color to the same as background.
![image](https://user-images.githubusercontent.com/21971068/233709313-54e19712-13a9-49cf-a120-39ca7f30a483.png)
Forth color is what is used for it.

**Comment by ghost** [comment]
This is fixed in the current BETA version. Just tested if for myself.

**Comment by ghost** [comment]
Close
